config/packages/security.yaml

```diff
@@ -65,215 +65,215 @@ security:
     role_hierarchy:
         ROLE_OAUTH2_WRITE:
             [
-                "ROLE_OAUTH2_ENTRY:CREATE",
```

Comment:
feel like we're going to be going back and forth a lot on this `'` vs `"` thing :laughing:

Reply:
haha yes.. it's final now `.editconfig` says single quote for all YAML files!